### PR TITLE
Create Generate_Pythagorean_Triplet.c

### PR DESCRIPTION
**Pull request by Harikrishnaa3131**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by iamrajiv — COMMENTED** [review]
@Harikrishnaa3131 Add sample input-output, space complexity and time complexity. Also, update the README.

File: C/math/Generate_Pythagorean_Triplet.c [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    int limit;
+    printf("Enter the limit:\n");
+    scanf("%d",&limit);
+    pythagoreanTriplets(limit);
+    return 0;
+}
+
+
+
+Sample Input:
```
Comment (atarax665) [inline C/math/Generate_Pythagorean_Triplet.c:37]: Add time/space complexity too. This whole clock should be in commented form.
i.e- 
```
/*
Time Complexity : 
Space Complexity : 

Sample I/O : 

Input : 

Output : 

*/
```

File: C/math/Generate_Pythagorean_Triplet.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Enter the limit:
+10
+Ouput:
+3 4 5
+8 6 10
+
+This program finds the pythagorean triplet in the given limit
```
Comment (atarax665) [inline C/math/Generate_Pythagorean_Triplet.c:44]: This should be at top. Do not forget to comment this line using ```//```.

File: C-Plus-Plus/math/Product_of_fractions.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        cin>>num[i];   
+    }
+    cout<<"Enter the denominators\n";
+    for(i=0;i<n;i++)
+    {
+        cin>>den[i];
+    }
+    prodRed(n, num, den); 
+    return 0; 
+}
```
Comment (atarax665) [inline C-Plus-Plus/math/Product_of_fractions.cpp:44]: Add sample input/output and Time and space complexity of your program in commented form. Refer any merged PR for better insight.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/math/Product_of_fractions.cpp [hunk 4]
```diff
@@ -0,0 +1,44 @@
+#include <bits/stdc++.h> 
+using namespace std; 
+
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/math/Product_of_fractions.cpp:3]: also remove these redundant spaces

File: C/math/Generate_Pythagorean_Triplet.c [hunk 5]
```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+
+void pythagoreanTriplets(int limit) 
+{ 
+
```
Comment (vatsalkesarwani12) [inline C/math/Generate_Pythagorean_Triplet.c:5]: remove redundant spaces from everywhere.

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**